### PR TITLE
Fix ocp provider

### PR DESCRIPTION
**Pull request by oshoval**
Need to line up to latest changes of ocp provider

### CODE REVIEW

**Comment by oshoval** [comment]
/hold
need to make sure its fine, maybe it should be on quay, we need Quique suggestion here

File: cluster-up/cluster/ocp-4.3/provider.sh [hunk 1]
```diff
@@ -27,7 +27,7 @@ function up() {
         workers=1
     fi
     echo "Number of workers: $workers"
-    params="--random-ports --background --prefix $provider_prefix --master-cpu 6 --workers-cpu 6 --secondary-nics ${KUBEVIRT_NUM_SECONDARY_NICS} --registry-volume $(_registry_volume) --workers $workers --container-registry quay.io ${image}"
+    params="--random-ports --background --prefix $provider_prefix --master-cpu 6 --workers-cpu 6 --secondary-nics ${KUBEVIRT_NUM_SECONDARY_NICS} --registry-volume $(_registry_volume) --workers $workers --container-registry docker-registry.upshift.redhat.com kubevirtci/${image}"
```
Comment (qinqon) [inline cluster-up/cluster/ocp-4.3/provider.sh]: is at quay not at upshift, please remove this.

File: cluster-up/cluster/ocp-4.3/provider.sh [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-image="openshift-cnv/kubevirtci-ocp-4.3@sha256:446265401bff3e90548f34461f0814493f52489c82e4527844250f7a02dcbb7c"
+image="ocp-4.3@sha256:63bc365f97acab249b34a048825edd650973764355b3f232db7d74f7d08e6ea4"
```
Comment (qinqon) [inline cluster-up/cluster/ocp-4.3/provider.sh]: Let's just regenerate this with the prow jobs that have just merged, drop this for now.

**Comment by oshoval** [comment]
will do just the needed stuff in a clean one (fix the pull secret + update the hash according new publish but stay on quay)

**Pull request closed without merging**